scripts/test-lab.val.micro

```diff
@@ -18,30 +18,33 @@ export LAB_TAG=rkv_perf_lab                #todo: use it to tag all vms of perf
 export NUM_OF_SI=2 
 
 ## disk size
+export RKV_ROOT_DISK_VOLUME=32
 export SI_ROOT_DISK_VOLUME=32
 
 ## rkv service
-export RKV_INSTANCE_TYPE=t2.micro 
+export RKV_INSTANCE_TYPE=t2.micro
 export RKV_AMI=ami-0bf8f78223ea6f3f6       #hw-ami-rkv-test2
 export RKV_VM_NAME=${NAME_TAG}-rkv-lab-rkv
 export RKV_REGION=us-west-2
-export RKV_ROOT_DISK_VOLUME=32
```

Comment:
would removal of this line lead to default disk volume size? what size is it then?

Reply:
this is a mistake. will revert this line.

Reply:
this was actually move to line 21. I'll consolidate this.